### DISCUSSION TITLE
Why is teh URL Scheme om:// limited?

### DISCUSSION THREAD

**Discussion by BtStSoft** [organicmaps/organicmaps#681] [Q&A]
From the source code I see that the scheme om:// does not function the same way as "mapswithme", "mapsme" and "mwm" (see for instance DeepLinkHandler.swift line 61).
My app GCBuddy would like to add OrganicMaps as external map app for showing the location of a waypoint on an external offline map using the om:// scheme.
So it would be nice if om:// also would use the already available handler for routes, searches etc.
Or was this done on purpose?

**Comment by biodranik** [comment]
om:// was added to primarily replace ge0:// URL scheme. It would be great to apply it for mwm too, but this needs some coding.
We didn't have time to clean and polish old schemes yet, so may be it's the right time to start thinking about it :)

**Reply by BtStSoft** [reply]
My suggestion would be to keep the current handling of old schemes intact so OrganicMaps is a true replacement app for Maps.me. Presence of the OrganicMaps app can be detected by the om:// handler already using canOpenURL on iOS.
What I now do is detect the presence using om:// and actually search a location using the "mapswithme://map" url scheme.

**Reply by rtsisyk** [reply]
ge0:// should be handled by Organic Maps.
